src/app/pages/CpsSty/categoryMap/index.js

```diff
@@ -0,0 +1,20 @@
+const categoryType = category => {
+  const dictonary = {
+    News: 'ReportageNewsArticle',
+    Feature: 'ReportageNewsArticle',
+    Infographic: 'ReportageNewsArticle',
+    Interactive: 'ReportageNewsArticle',
+    'Newsbeat Investigates': 'ReportageNewsArticle',
+    Summary: 'ReportageNewsArticle',
+    'You what?': 'ReportageNewsArticle',
+    Explainer: 'BackgroundNewsArticle',
+    Analysis: 'AnalysisNewsArticle',
+    Opinion: 'OpinionNewsArticle',
+    'Advertiser Content': 'AdvertiserContentArticle',
+    Review: 'ReviewNewsArticle',
+    'Ask the Audience': 'AskPublicNewsArticle',
```

Comment:
Found one missing category there:
>'Quiz' = 'ReportageNewsArticle', 

You may want to check again yourself: https://github.com/bbc/simorgh/issues/5424#issue-560575200